source/process/regex-library.md

```diff
@@ -87,56 +88,58 @@ title: RegEx Library
 <blockquote>
 <details close>
 <summary>Abbreviations</summary>
-- **tables to ABBR 1**: convert tables to abbreviation lists<br>F: <code>&#60;tr&#62;\s&#42;&#60;td&#62;(.&#42;?)&#60;/td&#62;\s&#42;&#60;td&#62;(.&#42;?)&#60;/td&#62;\s&#42;&#60;/tr&#62;</code><br>R: <code>&#60;dt epub:type=&#34;glossterm&#34;&#62;&#60;dfn&#62;\1&#60;/dfn&#62;&#60;/dt&#62;&#60;dd epub:type=&#34;glossdef&#34;&#62;\2&#60;/dd&#62;</code>
+- **tables to ABBR 1**: convert tables to abbreviation lists<br>F: <code>&#60;tr&#62;\s\*&#60;td&#62;(.\*?)&#60;/td&#62;\s\*&#60;td&#62;(.\*?)&#60;/td&#62;\s\*&#60;/tr&#62;</code><br>R: <code>&#60;dt epub:type=&#34;glossterm&#34;&#62;&#60;dfn&#62;\1&#60;/dfn&#62;&#60;/dt&#62;&#60;dd epub:type=&#34;glossdef&#34;&#62;\2&#60;/dd&#62;</code>
 
-- **tables to ABBR 2**: after running tables to ABBR 1 use this regex to format the lists new lines<br>F: <code>&#60;dfn&#62;(.&#42;?)&#60;/dfn&#62;&#60;/dt&#62;&#60;dd epub:type=&#34;glossdef&#34;&#62;(.&#42;?)&#60;/dd&#62;</code><br>R: <code>\n            &#60;dfn&#62;\1&#60;/dfn&#62;\n          &#60;/dt&#62;\n          &#60;dd epub:type=&#34;glossdef&#34;&#62;\2&#60;/dd&#62;</code>
+- **tables to ABBR 2**: after running tables to ABBR 1 use this regex to format the lists new lines<br>F: <code>&#60;dfn&#62;(.\*?)&#60;/dfn&#62;&#60;/dt&#62;&#60;dd epub:type=&#34;glossdef&#34;&#62;(.\*?)&#60;/dd&#62;</code><br>R: <code>\n &#60;dfn&#62;\1&#60;/dfn&#62;\n &#60;/dt&#62;\n &#60;dd epub:type=&#34;glossdef&#34;&#62;\2&#60;/dd&#62;</code>
```

Comment:
The spaces for formatting have been removed... I wonder if that's intended.

Reply:
I added the correct spaces back in but reviewing with live-server shows that not all the spaces are rendering properly. I will look into this more.